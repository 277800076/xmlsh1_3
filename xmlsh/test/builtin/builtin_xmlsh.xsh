# simple tests of xmlsh builtin command
# cannot test interactive subshells in a script

echo In parent shell
xmlsh -c 'echo In sub shell ; exit 1'
echo Exit value is $?
exit 0