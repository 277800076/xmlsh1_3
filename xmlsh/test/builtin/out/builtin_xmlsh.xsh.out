In parent shell
In sub shell
Exit value is 1
